src/llscheck.lua

```diff
@@ -29,10 +29,38 @@ local SeverityColor = {
    "white dim",
 }
 
+---Create a uniform isatty function.
+---@return fun(): boolean
+local function get_isatty()
+   local system_loaded, system = pcall(require, "system")
+   if system_loaded then
```

Comment:
`isatty` was added quite recently to `system`. We can add a condition `and system.isatty` to make sure isatty is defined in case an older version is installed.